api/v1/aerospikecluster_validating_webhook.go

```diff
@@ -1346,11 +1347,22 @@ func validateSecurityContext(
 func validateAerospikeConfigUpdate(
 	aslog logr.Logger, incomingVersion, outgoingVersion string,
 	incomingSpec, outgoingSpec, currentStatus *AerospikeConfigSpec,
+	podStatus map[string]AerospikePodStatus,
 ) error {
 	aslog.Info("Validate AerospikeConfig update")
 
+	isSecurityEnabledPodExist := false
+
+	for pod := range podStatus {
+		if podStatus[pod].IsSecurityEnabled {
+			isSecurityEnabledPodExist = true
+			break
+		}
+	}
+
```

Comment:
I think its better to move this code inside `validateSecurityConfigUpdate` func

Reply:
did not want to pass podStatus map unnecessarily, hence passing `security` related info only.